sway/desktop/output.c

```diff
@@ -1076,6 +1091,16 @@ void handle_output_layout_change(struct wl_listener *listener,
 	update_output_manager_config(server);
 }
 
+void handle_gamma_control_set_gamma(struct wl_listener *listener, void *data) {
+	struct sway_server *server =
+		wl_container_of(listener, server, gamma_control_set_gamma);
+	const struct wlr_gamma_control_manager_v1_set_gamma_event *event = data;
+
+	struct sway_output *output = event->output->data;
+	output->gamma_lut_changed = true;
```

Comment:
Was this state not meant to be cleared at some point?

Reply:
Indeed so. https://github.com/swaywm/sway/pull/7634